File: src/main/java/com/ploy/bubble_server_v3/domain/auth/presentation/dto/request/LoginRequest.java
package com.ploy.bubble_server_v3.domain.auth.presentation.dto.request;

import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotBlank;

public record LoginRequest(
        @Schema(description = "이메일")
        @NotBlank String email,
        @Schema(description = "비밀번호")
        @NotBlank String password,
        @Schema(description = "기기 토큰")
        @NotBlank String deviceToken
        ) {
}

File: src/main/kotlin/com/ploy/bubble_server_v2/domain/user/model/dto/request/LoginRequest.kt
package com.ploy.bubble_server_v2.domain.user.model.dto.request

import io.swagger.v3.oas.annotations.media.Schema
import lombok.Getter
import lombok.Setter

@Getter
@Setter
class LoginRequest {
    @Schema(description = "이메일")
    private val email: String? = null

    @Schema(description = "패스워드")
    private val password: String? = null
}